### PR TITLE
Delete-events-blog

### PR DESCRIPTION
**Pull request by MaxineCl**
### Trello card
https://trello.com/c/ZpA44wSR/5696-remove-and-redirect-events-blog-post

### Context
We have an old blog post on reasons to attend Get Into Teaching events https://getintoteaching.education.gov.uk/blog/5-reasons-to-attend-a-get-into-teaching-event.

We want to repurpose any useful info from this post and include it in the about events page and then delete the blog post. https://getintoteaching.education.gov.uk/events/about-get-into-teaching-events

### Changes proposed in this pull request

- add any useful info into the about events page
- delete the events blog post
- set up a redirect and redirect any pages that link to the blog post to the about events page

### CODE REVIEW

File: app/views/teaching_events/about_git_events.html.erb [hunk 1]
```diff
@@ -10,26 +10,30 @@
...
 
-    <p>These relaxed and informative events give you the opportunity to get expert advice and talk to teachers and local training providers.</p>
   </section>
 
   <section class="col col-720">
     <h3 class="heading-m">At our Get Into Teaching events</h3>
 
-    <p>Drop in any time during an event for presentations, one-to-one chats or to meet others like you who are thinking of stepping into the classroom.</p>
+    <p>Our advisers can help you understand the qualifications you need and the range of different ways you can train.</p>
+
```
Comment (gemmadallmandfe) [inline app/views/teaching_events/about_git_events.html.erb:23]: Looks like there is a linting issue on this line?

File: config/redirects.yml [hunk 2]
```diff
@@ -337,7 +337,7 @@ redirects:
   "/urgent-call-for-qualified-teachers": "/returning-to-teaching"
 
   # Blog pages
-  "/blog/5-reasons-to-attend-a-train-to-teach-event": "/blog/5-reasons-to-attend-a-get-into-teaching-event"
```
Comment (gemmadallmandfe) [inline config/redirects.yml:340]: I think we need to leave this redirect in but just change the destination, in case there are any old links to it ie

"/blog/5-reasons-to-attend-a-train-to-teach-event": "/events/about-get-into-teaching-events"

(We only renamed the blog last year when the event names changed)

File: config/redirects.yml [hunk 3]
```diff
@@ -337,7 +337,7 @@ redirects:
...
+  "/blog/5-reasons-to-attend-a-get-into-teaching-event" : "/events/about-get-into-teaching-events"
   "/blog/teachers-pension-scheme": "/is-teaching-right-for-me/what-pension-does-a-teacher-get"
```
Comment (gemmadallmandfe) [inline config/redirects.yml:341]: We should probably also add a redirect for the blog tag that has been deleted, for completeness:

FROM /blog/tag/get-into-teaching-events
TO /events/about-get-into-teaching-events

File: app/views/content/train-to-be-a-teacher/if-you-dont-have-a-degree.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ description: |-
   Explore how you can get into teaching without a degree by taking an undergraduate course to gain qualified teacher status (QTS) alongside a degree.
 related_content:
     Is teaching right for me? : "/is-teaching-right-for-me"
-    5 reasons to attend a Get Into Teaching event : "/blog/5-reasons-to-attend-a-get-into-teaching-event"
+    Get Into Teaching events what to expect : "/events/about-get-into-teaching-events"
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/if-you-dont-have-a-degree.md:8]: I think we should maybe delete this Related content link altogether - not sure that the events offer much for people without a degree / not studying for one?

File: app/views/teaching_events/about_git_events.html.erb [hunk 5]
```diff
@@ -10,26 +10,30 @@
...
+    <p>Our advisers can help you understand the qualifications you need and the range of different ways you can train.</p>
+    
+    <p>Drop in any time during an event for presentations, one-to-one chats or to meet people like you who are thinking of becoming a teacher.</p>
 
     <p>You can:</p>
 
     <ul>
-      <li>discover if teaching is right for you</li>
+      <li>speak to current teachers and get an insight into what life as a teacher is really like</li>
       <li>talk to training providers about courses they offer</li>
```
Comment (gemmadallmandfe) [inline app/views/teaching_events/about_git_events.html.erb:30]:
```suggestion
      <li>talk to training providers about the courses they offer</li>
```

File: app/views/teaching_events/about_git_events.html.erb [hunk 6]
```diff
@@ -10,26 +10,30 @@
...
+    <p>These relaxed and informative events provide a one-stop shop of information and support. They'll give you the opportunity to get expert advice and talk to teachers and local training providers.</p>
 
-    <p>These relaxed and informative events give you the opportunity to get expert advice and talk to teachers and local training providers.</p>
   </section>
 
   <section class="col col-720">
     <h3 class="heading-m">At our Get Into Teaching events</h3>
 
-    <p>Drop in any time during an event for presentations, one-to-one chats or to meet others like you who are thinking of stepping into the classroom.</p>
+    <p>Our advisers can help you understand the qualifications you need and the range of different ways you can train.</p>
```
Comment (gemmadallmandfe) [inline app/views/teaching_events/about_git_events.html.erb:22]: I think this line would be better after the 'drop in' line? We really want to emphasise the `drop in at any time` bit - came out of some event UR

Or alternatively add a couple more bullets:
* find out about the qualifications you need and the different ways you can train
* meet our advisers and ask them any questions you have about becoming a teacher

(you can word this better!)

File: app/views/teaching_events/about_git_events.html.erb [hunk 7]
```diff
@@ -10,26 +10,30 @@
...
+    
+    <p>Drop in any time during an event for presentations, one-to-one chats or to meet people like you who are thinking of becoming a teacher.</p>
 
     <p>You can:</p>
 
     <ul>
-      <li>discover if teaching is right for you</li>
+      <li>speak to current teachers and get an insight into what life as a teacher is really like</li>
```
Comment (gemmadallmandfe) [inline app/views/teaching_events/about_git_events.html.erb:29]:
```suggestion
      <li>speak to current teachers and find out what life as a teacher is really like</li>
```
can we shorten this a bit? you might do this better

File: app/views/teaching_events/about_git_events.html.erb [hunk 8]
```diff
@@ -10,26 +10,30 @@
...
       <li>talk to training providers about courses they offer</li>
+      <li>learn what funding is available</li>
       <li>find out how to apply for teacher training</li>
-      <li>explore what funding is available</li>
+      <li>make connections and keep in touch through our <a href="https://www.facebook.com/groups/1357146377672255)">Aspiring Teacher Forum</a> or <a href="https://www.facebook.com/groups/CareerChangetoTeaching">Career Change to Teaching</a> Facebook groups</li>
```
Comment (gemmadallmandfe) [inline app/views/teaching_events/about_git_events.html.erb:33]: I wonder whether this line is appropriate (I know it is in the blog) - let's discuss!

File: app/views/teaching_events/about_git_events.html.erb [hunk 9]
```diff
@@ -10,26 +10,30 @@
   </section>
 
   <section class="col col-720">
-    <p>Whether you're ready to start your career in teaching or just curious, we can answer your questions at a Get Into Teaching event.</p>
+    <p>We understand everyone’s journey into the classroom is different and your questions may be unique to your circumstances. But whether you're ready to start your career in teaching, or just curious, we can answer your queries at a Get Into Teaching event.</p>
```
Comment (gemmadallmandfe) [inline app/views/teaching_events/about_git_events.html.erb:13]:
```suggestion
    <p>We understand everyone’s journey into the classroom is different. But whether you're ready to start your career in teaching, or just curious, we can answer your queries at a Get Into Teaching event.</p>
```
Do we need the bit about circumstances? Feels a bit wordy on this page?

File: app/views/teaching_events/about_git_events.html.erb [hunk 10]
```diff
@@ -93,18 +97,3 @@
       <% end %>
     </section>
   </div>
-
-  <section class="col col-full-page what-theyre-saying">
```
[the remark this answers is not in the document]
Comment (MaxineCl) [reply app/views/teaching_events/about_git_events.html.erb:97]: I wasn't sure it added much as it's right at the bottom of the page. But happy to add it back in.

File: app/views/teaching_events/about_git_events.html.erb (hunk 1 above)
Comment (MaxineCl) [reply app/views/teaching_events/about_git_events.html.erb:21]: I've deleted this bit now so hopefully it's fixed it.

File: config/redirects.yml [hunk 11]
```diff
@@ -337,7 +337,7 @@ redirects:
   "/urgent-call-for-qualified-teachers": "/returning-to-teaching"
 
   # Blog pages
-  "/blog/5-reasons-to-attend-a-train-to-teach-event": "/blog/5-reasons-to-attend-a-get-into-teaching-event"
+  "/blog/5-reasons-to-attend-a-train-to-teach-event": "/events/about-get-into-teaching-events"
```
Comment (gemmadallmandfe) [inline config/redirects.yml:340]: sorry - I think my previous comment was confusing - we need **both** redirects

"/blog/5-reasons-to-attend-a-train-to-teach-event": "/events/about-get-into-teaching-events"
"/blog/5-reasons-to-attend-a-get-into-teaching-event": "/events/about-get-into-teaching-events"

Comment (MaxineCl) [reply config/redirects.yml:340]: Ah sorry. No problem. I'll update that asap.

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me
